website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+
+Below the Git Actions Button, all changes to files since the last commit will be listed in the **Changes** section. Clicking on any change will open the Git Diff View of the selected file, allowing the user to see the inline changes.
+
+#### File Explorer 
+The File Explorer shows the filetree of your repository. You can:
+* Click on any file in the filetree to open the file in the File Editor. 
+* Click and drag files between directories to move files. 
+* Right click a file to access the sub-menu options like copy file name, copy as `ref`, rename, delete, and more.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-file-explorer-menu.jpg" width="30%" title="Right click files to access sub-menu options"/>
+
+#### Command bar
+The Command bar is where [dbt commands](/reference/dbt-commands) are invoked in the IDE. Invoking a command will open the Invocation History Drawer to show the associated logs.
+
+#### IDE Status button
+The IDE Status Button reports the current status of the IDE. Anytime there is a status error, or if there is a dbt code error that prevents the project from parsing, this button will turn red and say Error. Clicking on this button brings up the [IDE Status Modal](#IDE-Status-Modal)
+
+
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-options-menu.jpg" width="85%" title="Click on the IDE status button to view the options menu"/>
+
+#### Create New File button
+This button allows the user to create a new file in their project. 
+
+## Editing 
+
+The layout of the IDE editing page is divided into the following areas, and more features.
+
+A &mdash; [File Editor](#file-editor)<br />
+B &mdash; [Interaction bar](#interaction-bar)<br />
+C &mdash; [Preview button](#preview-button)<br />
+D &mdash; [Compile button](#compile-button)<br />
+E &mdash; [Build button](#build-button)<br />
+F &mdash; [Results subtab](#results-subtab)<br />
+G &mdash; [Compiled code subtab](#compiled-code-subtab)<br />
+H &mdash; [Lineage subtab](#lineage-subtab)<br />
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-editing.jpg" width="100%" title="The Cloud IDE editing components"/>
+
+#### File Editor
+The File Editor is the region where users edit code. This region is broken out by tabs for all files that have been opened. Unsaved Files are marked with a blue dot icon in the tab view. 
+
+The editor has a **Format** button enabled in this area for `{{  filetypes  }}` files, and will use your selected formatter (sqlfmt or sqlfluff) to edit the file contents. 
+
```

Comment:
You should call out that this (which I would call the Console) is very contextual! Opening different types of files will provide different Console Tabs and interactions